client/src/components/Donation/donation-modal.tsx

```diff
@@ -79,8 +79,9 @@ const RenderIlustration = ({
 }) => {
   const showModalBears = useFeature('show-modal-bears').on;
   if (showModalBears) {
-    if (recentlyClaimedBlock !== null) return <BearBlockCompletion />;
-    else return <BearProgressModal />;
+    if (recentlyClaimedBlock !== null)
+      return <BearBlockCompletion className='donation-icon' />;
+    else return <BearProgressModal className='donation-icon' />;
   } else if (recentlyClaimedBlock !== null) {
     return <Cup className='donation-icon' />;
   } else {
```

Comment:
I know I'm guilty of worse (and it's a little out of scope), but the inconsistency in the brackets makes this hard to read.

How about:
```jsx
  if (showModalBears) {
    return recentlyClaimedBlock ? (
      <BearBlockCompletion className='donation-icon' />
    ) : (
      <BearProgressModal className='donation-icon' />
    );
  } else {
    return recentlyClaimedBlock ? (
      <Cup className='donation-icon' />
    ) : (
      <Heart className='donation-icon' />
    );
  }
```

Since it's either truthy or `null` the explicit `null` checking isn't necessary.

Reply:
done.